src/antidote_pb_txn.erl

```diff
@@ -71,7 +71,7 @@ process(#fpbatomicupdatetxnreq{ops = Ops}, State) ->
     end;
 
 process(#fpbsnapshotreadtxnreq{ops = Ops}, State) ->
-    lager:info("Testing txn interface.. Received snapshot read request  ~p", [Ops]),
```

Comment:
Why don’t you then completely remove it?
Or replace it by a less expensive error message?



> On 19.05.2015, at 17:03, marsleezm <notifications@github.com> wrote:
> 
> In src/antidote_pb_txn.erl <https://github.com/SyncFree/antidote/pull/121#discussion_r30608554>:
> 
> > @@ -71,7 +71,7 @@ process(#fpbatomicupdatetxnreq{ops = Ops}, State) ->
> >      end;
> >  
> >  process(#fpbsnapshotreadtxnreq{ops = Ops}, State) ->
> > -    lager:info("Testing txn interface.. Received snapshot read request  ~p", [Ops]),
> Because this outputs a message every time a transaction is received.. This will have severe performance penalty.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/121/files#r30608554>.
>